mlflow/sklearn/__init__.py

```diff
@@ -945,6 +970,10 @@ def patched_fit(original, self, *args, **kwargs):
                     FLAVOR_NAME, class_def, func_name, patched_fit, manage_run=True,
                 )
 
+    from sklearn import metrics
+    for metric_method in ['accuracy_score', 'r2_score']:
+        safe_patch(FLAVOR_NAME, metrics, metric_method, patched_metric_api, manage_run=False)
+
```

Comment:
TODO: 
* support all metric APIs.
* seems this patch break the model.score() api, to be investigate.